libs/shared/src/consts/providers/provider.enum.ts

```diff
@@ -89,7 +89,7 @@ export enum SmsProviderIdEnum {
   Bandwidth = 'bandwidth',
   MessageBird = 'messagebird',
   Simpletexting = 'simpletexting',
-  AzureSms = 'azureSms',
+  AzureSms = 'azure-sms',
```

Comment:
In `@novu/azure-sms` identifier is: `azure-sms`:

https://github.com/parveshsaini/novu/blob/8bffdb1b6dfd1a7ac3d73d88dcffa954be73f6bd/providers/azure-sms/src/lib/azure-sms.provider.ts#L9C1-L12